capabilities/go.mod

```diff
@@ -0,0 +1,78 @@
+module github.com/smartcontractkit/chainlink-framework/capabilities
```

Comment:
Please add a readme with short description and instructions, also point to EVM and Aptos implementations. Thx!